src/ConnectedMode/Binding/IBindingProcessFactory.cs

```diff
@@ -61,12 +62,16 @@ public IBindingProcess Create(BindCommandArgs bindingArgs, bool isFirstBinding)
             var bindingConfigProvider = CreateBindingConfigProvider();
             var solutionBindingOp = new SolutionBindingOperation();
 
-            return new BindingProcessImpl(bindingArgs,
```

Comment:
The `BindingProcessImpl` is directly created by a MEF-imported factory (until recently, it was created inline in one of the binding classes).

To limit the scope of this PR, I've made this class directly the new downloader class too. We'll need to make the downloader MEF-creatable so we can re-use it from the timed updater etc.
However, to do that we'll also need to make some other classes MEF-creatable e.g. `CompositeBindingConfigProvider` -> will done in a separately.